### ISSUE TITLE
Performance comparison of ssd_mobilenet_v1_coco, faster_rcnn_inception_v2_coco, and rfcn_resnet101_coco 

### ISSUE THREAD

**Issue by GuomingLi565** [tensorflow/models#6654]
Hello,

I used the tensorflow API models of ssd_mobilenet_v1_coco (SSD), faster_rcnn_inception_v2_coco (faster RCNN), and rfcn_resnet101_coco (RFCN) to trained the customized detectors. And they worked pretty well. The accuracies of detecting desired objects based on these three models were  23% for SSD, 96% for faster RCNN, and 74% for RFCN. I am wondering why the SSD and RFCN had such low accuracy for my detection purpose. Is there anyone having some comments on this?

**Comment by rootkitchao** [comment]
This is caused by the situation of the dataset you are using and the hyperparameters you set.First, when the SSD does not use Focal loss, the detection accuracy is significantly lower than that of the second-stage detector. And Mobilenet V1 is a less accurate backbone.Technically rfcn resnet-101 will be more accurate than faster rcnn inception v2.Actual situation will be affected by training.

**Comment by GuomingLi565** [comment]
Hello William,

Thank you for your comments. I have one more question that how could I
avoid overfitting for the model training. Based on the observation, the
RFCN model could wrongly detect more objects than the ground truth, which
may compromise the accuracy of the RFCN model. Do you have any thoughts on
that?

Best wishes,
Guoming

------------------------------------------------------------------------------------------------------
Guoming Li, Ph.D. Student, Research Assistant
Department of Agricultural and Biological Engineering
Mississippi State University
Email: gl565@msstate.edu
ResearchGate webpage: https://www.researchgate.net/profile/Guoming_Li2
------------------------------------------------------------------------------------------------------

**Comment by rootkitchao** [comment]
> Hello William,
> 
> Thank you for your comments. I have one more question that how could I
> avoid overfitting for the model training. Based on the observation, the
> RFCN model could wrongly detect more objects than the ground truth, which
> may compromise the accuracy of the RFCN model. Do you have any thoughts on
> that?
> 
> Best wishes,
> Guoming
> …

  
To reduce overfitting, you can try multiscale training and [dropblock](https://arxiv.org/pdf/1810.12890.pdf).But this is not necessarily caused by overfitting, it may be because the network is not fully trained or the data set is too small.
